src/kademlia/item.cpp

```diff
@@ -69,7 +68,6 @@ namespace
 			left = out.size() - (ptr - out.data());
 			std::memcpy(ptr, salt.data(), (std::min)(salt.size(), left));
 			ptr += (std::min)(salt.size(), left);
-			left = out.size() - (ptr - out.data());
```

Comment:
but it's updated two lines below, rendering this update null. Do you want it back regardless?